chat-commands.js

```diff
@@ -2403,10 +2406,14 @@ exports.commands = {
 				this.add(`|unlink|${hidetype}${targetUser.prevNames[prevName]}`);
 			}
 		} else {
-			this.addModAction(`${name}'s messages were cleared from ${room.title} by ${user.name}.`);
+			if (targetUser) {
```

Comment:
This still doesn't need to be changed. What was wrong with the old code? Can you show me the error message?